integration/examples/jib-sync/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
Here too, why not the latest, 8.11.1?

Once you've changed this file, can you run `./gradlew wrapper` which  will update a few additional files to the latest versions. (If you don't do this, it's going to redownload gradle 8.10 in every integration test run... once you run that command, it'll check the new version into git.)